### ISSUE TITLE
Can't send SMS to PhoneMember ClientType

### ISSUE THREAD

**Issue by GoogleCodeExporter** [msndevs/msnp-sharp#72]
```
What steps will reproduce the problem?
1. create a PhoneMember in MSN by adding a contact with only a phone number
2. Try to send the contact a mobile message with 
messenger.Nameserver.SendMobileMessage(contact, string);
3. An exception returns because no mobile device is enabled.

What is the expected output? What do you see instead?
When I do this with windows live messenger, I'm able to send an SMS to the 
phone. When I try to do the same thing with the MSNPSharp, I get an 
exception. The only way I can send SMS is by having the user register 
their phone using the mobile phone registration with windows live first. 
In other words it has to be with a passport account, not just a phone 
number account.

What version of the product are you using? On what operating system?
I downloaded the most recent version about a week ago. around 1/16/08.

Please provide any additional information below.

I haven't tried adding the contact through MSNPSharp yet. I'm doing all of 
my contact setup in windows live messenger, then launching my project and 
trying to send an SMS message.

```

Original issue reported on code.google.com by `robertmo...@gmail.com` on 22 Jan 2009 at 9:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `freezing...@gmail.com` on 23 Jan 2009 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
Hi,

We need your msn account, password and the phone number you sending sms.
Or, network log from official wlm... You sent sms messages with official wlm 
successfuly.

Maybe, Contact.MobileAccess maybe obsoloted in MSNP15 and we are using this 
property
before sent.
```

Original comment by `hepha...@gmail.com` on 23 Jan 2009 at 2:25

**Comment by GoogleCodeExporter** [comment]
```
MSNP18 uses UUM command to send mobile message or OIM messages:

UUM 15 tel:+862341234567 4 1 89 (I think 4: ClientType.PhoneMember=4)
MIME-Version: 1.0
Content-Type: text/plain; charset=iso-8859-1
Dest-Agent: mobile

hi
```

Original comment by `hepha...@gmail.com` on 23 Jan 2009 at 3:01

**Comment by GoogleCodeExporter** [comment]
Original comment by `hepha...@gmail.com` on 23 Jan 2009 at 3:01

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Could you upgrade your code to:

http://msnp-sharp.googlecode.com/svn/branches/MSNPSHARP_25_STABLE
AND
http://msnp-sharp.googlecode.com/svn/trunk

Please report us it is working in 2 versions.


```

Original comment by `hepha...@gmail.com` on 23 Jan 2009 at 4:15
- Changed state: **Fixed**
- Added labels: **Milestone-Release2.5.4**

**Comment by GoogleCodeExporter** [comment]
```
sorry, I'm delayed in responding. I used wireshark to look at the difference 
between 
what msnsharp sent out and what MSN sent out. The sendmobilemessage function 
checked 
for mobileaccess from the contact. My contact didn't exist, so this failed. 
When I 
changed that in MSNSharp, it sent a message, but it failed. I found that it was 
sending the message to the email account instead of the phone number. I changed 
it 
so that it sent to the phone number with "tel:" in front of it, and it worked 
great.
In the below code, SendTo is a mobile phone number structured like 
+19995556666. I 
haven't tried newer versions. Perhaps it's working now. Thanks.

MobileMessage bodyMessage = new MobileMessage();
                bodyMessage.CallbackDeviceName = string.Empty;
                bodyMessage.CallbackNumber = string.Empty;
                bodyMessage.Receiver = SendTo;
                bodyMessage.Text = Message;

                // create a NSPayLoadMessage to transport it
                MSNPSharp.Core.NSPayLoadMessage nsMessage = new 
MSNPSharp.Core.NSPayLoadMessage("PGD", new string[] { "tel:" + 
bodyMessage.Receiver, "1" }, Encoding.UTF8.GetString(bodyMessage.GetBytes()));

                // and send it
                messenger.Nameserver.MessageProcessor.SendMessage(nsMessage);
```

Original comment by `robertmo...@gmail.com` on 29 Jan 2009 at 3:07

**Comment by GoogleCodeExporter** [comment]
```
You guys rock! any of you got trace for an incoming sms? i mean the reply from 
a 
mobile back to the msn window?
```

Original comment by `zeeshane...@gmail.com` on 30 Jan 2009 at 10:10

**Comment by GoogleCodeExporter** [comment]
```
Hi man, I haven't try, but you can send email to hephappy@gmail.com to ask 
Ethem, he 
is the expert of MSN SMS.
```

Original comment by `issues.m...@gmail.com` on 31 Jan 2009 at 11:55

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the reply :)
```

Original comment by `zeeshane...@gmail.com` on 31 Jan 2009 at 7:59

**Comment by GoogleCodeExporter** [comment]
```
zeeshanejaz, did you ever find out that information on incoming sms's? I'm 
currently 
trying to receive response sms messages from my phone, but I can't seem to find 
any 
events to monitor. Did you learn anything?
```

Original comment by `robertmo...@gmail.com` on 13 Feb 2009 at 12:18
